src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -0,0 +1,101 @@
+            cellRenderer: ({ rowIndex }) => {
+              const { shardIndex } = callObjects[rowIndex]
+              return shardIndex >= 0 ? shardIndex : 'N/A'
+            }
+          }, {
+            size: { basis: 100, grow: 0 },
+            headerRenderer: () => 'Attempt',
+            cellRenderer: ({ rowIndex }) => {
+              const { attempt } = callObjects[rowIndex]
+              return attempt
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Status',
+            cellRenderer: ({ rowIndex }) => {
+              const { executionStatus } = callObjects[rowIndex]
+              return h(TooltipCell, [makeCromwellStatusLine(executionStatus)])
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Start',
+            cellRenderer: ({ rowIndex }) => {
+              const { start } = callObjects[rowIndex]
+              return h(TooltipCell, [start ? Utils.makeCompleteDate(start) : 'N/A'])
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'End',
+            cellRenderer: ({ rowIndex }) => {
+              const { end } = callObjects[rowIndex]
+              return h(TooltipCell, [end ? Utils.makeCompleteDate(end) : 'N/A'])
+            }
+          },
+          {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Call Caching Result',
+            cellRenderer: ({ rowIndex }) => {
+              const { callCaching: { effectiveCallCachingMode, result } = {} } = callObjects[rowIndex]
+              if (effectiveCallCachingMode === 'ReadAndWriteCache' || effectiveCallCachingMode === 'ReadCache') {
+                return result ? h(TooltipCell, [result]) : div({ style: { color: colors.dark(0.7) } }, ['No Information'])
+              } else if (effectiveCallCachingMode === 'WriteCache') {
+                return div({ style: { color: colors.dark(0.7) } }, ['Written but not read'])
+              } else {
+                return div({ style: { color: colors.dark(0.7) } }, [effectiveCallCachingMode])
+              }
+            }
+          },
+          {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Links',
+            cellRenderer: ({ rowIndex }) => {
+              const { failures, shardIndex: index, attempt } = callObjects[rowIndex]
+              const failureCount = _.size(failures)
+              return [
+                failures && h(Link, {
+                  style: { marginLeft: '0.5rem' },
+                  onClick: () => setFailuresModalParams({ index, attempt, failures })
+                }, [
+                  div({ style: { display: 'flex', alignItems: 'center' } }, [
+                    icon('warning-standard', { size: 18, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+                    `${failureCount} Error Message${failureCount > 1 ? 's' : ''}`
```

Comment:
Marketing hat: a message stating that e.g. an attempt was preempted is not an error or at all unexpected.
```suggestion
                    `${failureCount} Message${failureCount > 1 ? 's' : ''}`
```

Reply:
I think the UI is valid in carrying this forward as what the metadata is describing as an error. IMO we could perhaps look to instead move "you got preempted" out of the `failures` section of metadata and into a new `warnings` or `messages` section? (I can think of other possible non-error-level messages that we might want to be able to highlight to users if the UI had such a section).

Reply:
Or maybe we could describe them as "errors" if the call has failed, but just "messages" if it succeeded?

Reply:
IMO the UI doesn't have to match the underlying data structure, it should serve the user as clearly as possible and may need to add/remove meaning from the data

Reply:
My reluctance is that by just calling them "messages" here that real errors would also just be labeled as "Messages" which seems worse to me thanlumping preemption messages in with the errors. 

That said - I'd be happy to see a little logic in a follow-up PR around whether to call individual messages "errors" vs "messages"

Reply:
Also happy to be overridden if you still feel strongly that you still want me to change it. I guess I'm reluctant but not totally fixed in my opinion